data/wpa_supplicant/wpa_supplicant_test.sh

```diff
@@ -116,9 +117,12 @@ if systemctl is-active wickedd >& /dev/null; then
     echo "Restarting wicked ... "
     systemctl restart wicked
     sleep 20
-    systemctl status wicked
+    systemctl status wicked | cat
     echo "starting wicked ... "
-    wicked --debug all ifup wlan1 --timeout 60 > wicked.log 2>&1
+    # wicked ifup might terminate with setup-in-progress (rc=162), so check the link manually
+    wicked --debug ifup wlan1 --timeout 60 > wicked.log 2>&1 || true
+    sleep 30
+    wicked ifstatus wlan1 | grep 'link' | grep 'state up\|state device-up'
```

Comment:
I think we should judge from wicked perspective.

```suggestion
    wicked --log-level debug --debug all ifup wlan1 --timeout 90 2>&1 | tee wicked.log
```

Reply:
Please see the comment just above this command:

    # wicked ifup might terminate with setup-in-progress (rc=162), so check the link manually

This is one of the reasons why this test occasionally fails. AFAICS `wicked` terminates prematurely with rc=162, but the link is setup correctly. In this state the test fails because we have `-e` set.

Reply:
yes, thats why I increased from 60 to 90 the timeout as you also do this `60 + sleep 30`. 
And I actually wonder why this failing. Even 60 seconds should be enough in all circumstances here.

Reply:
In my test runs it never waited for even the 60 seconds but terminated after just a couple of them, so increasing the timeout alone is not sufficient.

Reply:
oh, that is something I didn't expect.  I need to check it then lets first go with your change to unblock the tests. 
Wicked wlan is tested in wicked-testsuite https://github.com/os-autoinst/os-autoinst-distri-opensuse/tree/master/tests/wicked/wlan/sut

Reply:
I was to fast... now I realized your words :)

```bash
wicked --debug ifup wlan1 --timeout 60
Bad debug facility "ifup"
wicked [options] <command> ...
```
This is what I get with your command. I guess it should be `--debug all` ...